### DISCUSSION TITLE
Is there any way to use it without using a ViewModel?

### DISCUSSION THREAD

**Discussion by OpenPachinko** [ME-MarvinE/XCalendar#151] [Q&A] [answered]
I am not making anything spectacular,
I'd like to use a quick and easy ViewModel without using it,
Is there any way to use it that way?

**Comment by ME-MarvinE** [comment, marked as answer]
Yes, do the same as you would using any other control. You can use x:Name on the CalendarView in the XAML and then change any of its properties from the code behind.

 But if you're still okay with using bindings (And don't want to redo parts of copy-pasted code from the sample app like DataTriggers), using the [Getting Started](https://github.com/ME-MarvinE/XCalendarGettingStarted) repo as an example:

1. Move `public Calendar<CalendarDay> MyCalendar { get; set; } = new Calendar<CalendarDay>();` from the ViewModel to the code behind for the page. 
2. Replace `BindingContext = new MainPageViewModel()` with `BindingContext = this` in the code behind of the page.

After that you should be able to use bindings as normal.
